textgrad/engine/openai.py

```diff
@@ -26,6 +27,7 @@ def __init__(
         model_string: str="gpt-3.5-turbo-0613",
         system_prompt: str=DEFAULT_SYSTEM_PROMPT,
         is_multimodal: bool=False,
+        base_url: str=None,
```

Comment:
can you add a param string explaining that base URL is currently used to support ollama?